boa3/internal/model/builtin/contract/nep17contract.py

```diff
@@ -0,0 +1,32 @@
+from typing import Dict
+
+from boa3.internal.model.builtin.interop.contract import ContractType
+
+from boa3.internal.model.method import Method
+
+
+class Nep17ContractClass(ContractType):
+    def __init__(self):
+        super().__init__()
+        
+        self._identifier = 'Nep17Contract'
+        
+        from boa3.internal.model.builtin.contract.nep17_interface_methods import (
+            Nep17InterfaceBalanceOfMethod,
+            Nep17InterfaceDecimalsMethod,
+            Nep17InterfaceSymbolMethod,
+            Nep17InterfaceTotalSupplyMethod,
+            Nep17InterfaceTransferMethod,
+        )
+
+        self._instance_methods: Dict[str, Method] = {
+            'balance_of': Nep17InterfaceBalanceOfMethod(self),
+            'decimals': Nep17InterfaceDecimalsMethod(self),
+            'symbol': Nep17InterfaceSymbolMethod(self),
+            'total_supply': Nep17InterfaceTotalSupplyMethod(self),
+            'transfer': Nep17InterfaceTransferMethod(self),
+        }
```

Comment:
since we're saving the interface identifier on the `Nep17InterfaceMethod` object, I think it's better to iterate over them instead of using the strings directly

```suggestion
        nep17_methods: List[Type[Nep17InterfaceMethod]] = [
            Nep17InterfaceBalanceOfMethod,
            Nep17InterfaceDecimalsMethod,
            Nep17InterfaceSymbolMethod,
            Nep17InterfaceTotalSupplyMethod,
            Nep17InterfaceTransferMethod,
        ]
        
        instance_methods: Dict[str, Method] = {}
        for method in nep17_methods:
            method_obj = method(self)
            instance_methods[method_obg.identifier] = method_obj
        
        self._instance_methods = instance_methods
```